Documentation/network/kubernetes/kubeproxy-free.rst

```diff
@@ -600,6 +600,15 @@ the extra hop, meaning, backends reply by using the service IP/port as a source.
 
 Another advantage in DSR mode is that the client's source IP is preserved, so policy
 can match on it at the backend node. In the SNAT mode this is not possible.
+
+.. warning::
+   Be aware that the client's source IP is **not** preserved by the ingress
+    (e.g. --set ingressController.enabled=true), so 
+    ``ciliumclusterwidenetworkpolicies.cilium.io``, ``ciliumnetworkpolicies.cilium.io`` 
```

Comment:
CCNP does support this https://docs.cilium.io/en/stable/network/servicemesh/ingress-and-network-policy/#gs-ingress-and-network-policy

Reply:
let me test this quickly on my prod cluster and come back to you